webhook_to_fedora_messaging/endpoints/models/service.py

```diff
@@ -0,0 +1,62 @@
+from abc import ABC
+from datetime import datetime
+from enum import Enum
+from typing import List, Optional
+
+from pydantic import BaseModel
+
+
+class ServiceType(str, Enum):
+    github = "github"
+
+
+class ServiceBase(BaseModel, ABC):
+    """
+    Base: Service
+    """
+
+    class Config:
+        from_attributes = True
+
+
+class ServiceExternal(ServiceBase):
+    uuid: str
+    name: str
+    type: ServiceType
+    desc: str
+    token: str
+    user_id: int
```

Comment:
It should probably be `username` here, no? Otherwise it'll be hard to match it to the associated user since we don't expose user IDs.

Reply:
Agreed but there is a lot of moving parts in this change. There are three ways with which the users are referenced in the project currently (via `id`, `uuid` and `username`), all of which are unique in nature. I suggest that we merge the progress we have made so far before we make the changes in the database to get rid of the `uuid` column so that all references are made with just `username`. Alternatively, I could make two database requests per `service` unit (i.e. one for the `service` unit itself and the other for getting the `user` unit from `id`) but that would be unoptimal.

Reply:
Agreed, let's change that later. For the record you don't need to make two database queries, that's what [selectinload](https://docs.sqlalchemy.org/en/20/orm/queryguide/relationships.html#sqlalchemy.orm.selectinload) is for (it's been removed now but we could put it back).